content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -154,6 +154,12 @@ The kubelet has the following default hard eviction thresholds:
 * `imagefs.available<15%`
 * `nodefs.inodesFree<5%` (Linux nodes)
 
+These default values of hard eviction thresholds will only be set if none 
+of the parameters is changed. If you changed the value of any parameter, 
+then the values of other parameters will not be inherited as the default 
+values and will be set to zero. In order to provide custom values, you 
+should provide all the thresholds respectively.
```

Comment:
Personally I'd pop this in a [note](https://kubernetes.io/docs/contribute/style/style-guide/#note) callout here as well, but that's not a critical change to make.